[3.3] Scrollable Tabs (#921)
telerik/blazor-docs: components/tabstrip/scrollable-tabs.md

---
title: Scrollable Tabs
page_title: TabStrip Scrollable Tabs
description: Scrollable Tabs in the TabStrip for Blazor.

> slug: tabstrip-scroll-tabs

Reviewer: scroll-tabs may be harder to remember and use without checking. Why not use scrollable-tabs, as the article title?

tags: telerik,blazor,tab,strip,tabstrip,scroll,tabs
published: True
position: 15
---

# TabStrip Scrollable Tabs

> By default, if the accumulative width or height of the tabs in the TabStrip exceed the width or height of the browser a browser-level scrollbar will appear. 

Reviewer: Consider a simpler version. It's "cumulative" by the way.
```suggestion
If the TabStrip dimensions exceed the width or height of the page, a browser-level scrollbar will appear. 
```

> To scroll the tabs only, set the `Scrollable` bool parameter of the TabStrip to `true` (defaults to `false`). You should also set the `Width` (for tabs with [Position]({%slug tabstrip-tabs-position%}) set to top and bottom) and `Height` (for tabs with [Position]({%slug tabstrip-tabs-position%}) set to left and right) to achieve the desired behavior.

Reviewer: Can be a list of 3 steps as well.

>caption Scroll the tabs only

<div class="skip-repl"></div>
````HorizontalScrolling
<TelerikTabStrip Scrollable="true"
                 Width="300px"
                 TabPosition="Telerik.Blazor.TabPosition.Top">

>     <TabStripTab Title="First">

Reviewer: Generate the tabs with a loop to make the example simpler.
